app/controllers/menus_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MenusController < ApplicationController
+  before_action :set_menu, only: [:show, :edit, :update]
```

Comment:
`only: [:show, :edit, :update]` => `only: %i[show edit update]`
`%i %w` for the win